src/styles/blocks/phone.scss

```diff
@@ -0,0 +1,50 @@
+@import "../utils/vars";
+@import "../utils/mixins";
+
+.phone {
+  width: 18px;
+  height: 18px;
+  order: 2;
+
+  @include for-tablet {
+    width: 24px;
+    height: 24px;
+  }
+
+  @include for-desktop {
+    &:hover + &__number {
+      width: 141px;
+      display: block;
+      margin-right: 12px;
+      margin-top: 4px;
+    }
+  }
+
+  &__ {
+    &number {
+      order: 1;
+
+      font-family: "Inter", sans-serif;
+      font-style: normal;
+      font-weight: 700;
+      font-size: 12px;
+      line-height: 16px;
+      letter-spacing: 2px;
+      color: $c-light-black;
+      text-decoration: none;
+
+      display: none;
+      width: 0;
+    }
+
+    &icon {
+      width: 100%;
+      height: 100%;
+
+      @include for-desktop {
+        width: 24px;
+        height: 100%;
```

Comment:
Change 'height: 100%;' to 'height: 24px;' to match the width value for a consistent sizing on desktop.